core/invoker/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -60,7 +60,7 @@ object Invoker {
       invokerContainerDns -> "",
       invokerContainerNetwork -> null,
       invokerUseRunc -> "true") ++
-      Map(invokerName -> null)
+      Map(invokerName -> "")
```

Comment:
invokerName should be validated as non-empty below, same as redisHost (it will only be validated when the id arg is not present)

Reply:
good point @tysonnorris. 
done